### PR TITLE
Add support for time as timestamp

### PR DESCRIPTION
**Pull request by cedk**
I use the log from [ii](https://tools.suckless.org/ii/) which switched to timestamp format for the log.

### CODE REVIEW

File: src/irclog2html/irclog2html.py [hunk 1]
```diff
@@ -119,6 +120,7 @@ class LogParser(object):
         r'(?:\d{4}-\d{2}-\d{2}T|\d{2}-\w{3}-\d{4} |\w{3} \d{2} |\d{2} \w{3} )?' # Optional date
         r'\d\d:\d\d(:\d\d)?' # Mandatory HH:MM, optional :SS
         r')\]? +') # Optional ], mandatory space
+    TIMESTAMP_REGEXP = re.compile(r'^(\d*) +')
```
Comment (mgedmin) [inline src/irclog2html/irclog2html.py]: I think it should be `\d+`, to require at least one digit.

File: src/irclog2html/irclog2html.py [hunk 2]
```diff
@@ -160,6 +162,13 @@ def __iter__(self):
             else:
                 time = None
 
+            if time is None:
+                m = self.TIMESTAMP_REGEXP.match(line)
+                if m:
+                    time = datetime.datetime.fromtimestamp(
+                        int(m.group(1))).isoformat()
+                    line = line[len(m.group(0)):]
```
Comment (mgedmin) [inline src/irclog2html/irclog2html.py]: The Travis failure looks to be caused by timezones.  Now I'm wondering if this should be `utcfromtimestamp()`, or if I should change my test harness to set `TZ=UTC`.

Comment (cedk) [reply src/irclog2html/irclog2html.py]: Indeed it makes sense to parse timestamp as UTC.

**Comment by mgedmin** [comment]
Thank you!

**Pull request merged**